packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1007,4 +1007,14 @@ whisperkit-cli transcribe --audio-path /path/to/audio.mp3
 # Or use your preferred model variant
 whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`,
 ];
+
+export const threedtopia_xl = (): string[] => [
+	`# Install from https://github.com/3DTopia/3DTopia-XL
+from threedtopia_xl.models import threedtopia_xl
+model = threedtopia_xl.from_pretrained("${model.id}")
+img = 'PATH_TO_Image.png'
+model.generate(cond=img)
+# refer to https://github.com/3DTopia/3DTopia-XL?tab=readme-ov-file#inference for inference
+# and https://github.com/3DTopia/3DTopia-XL/blob/main/app.py regarding usage`,
```

Comment:
```suggestion
	`from threedtopia_xl.models import threedtopia_xl

model = threedtopia_xl.from_pretrained("${model.id}")
model.generate(cond="path/to/image.png")
```

We usually tend to make the code snippet as minimal as possible. The link to https://github.com/3DTopia/3DTopia-XL will already be there if users want to know more.

Reply:
Sounds good